system-x/services/google/bigquery/src/main/java/software/tnb/google/cloud/bigquery/service/GoogleBigQuery.java

```diff
@@ -0,0 +1,68 @@
+package software.tnb.google.cloud.bigquery.service;
+
+import software.tnb.common.account.Accounts;
+import software.tnb.common.service.Service;
+import software.tnb.google.cloud.bigquery.validation.BigQueryValidation;
+import software.tnb.google.cloud.common.account.GoogleCloudAccount;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.api.gax.core.FixedCredentialsProvider;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auto.service.AutoService;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Base64;
+
+@AutoService(GoogleBigQuery.class)
+public class GoogleBigQuery implements Service {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GoogleBigQuery.class);
+
+    private GoogleCloudAccount account;
+    private BigQueryValidation validation;
+
+    private BigQuery bigQuery;
+
+    public GoogleCloudAccount account() {
+        if (account == null) {
+            account = Accounts.get(GoogleCloudAccount.class);
+        }
+        return account;
+    }
+
+    protected BigQuery bigQueryClient() throws IOException {
+        LOG.debug("Creating new Google BigQuery client");
+        BigQuery bigquery =
+            BigQueryOptions.newBuilder().setCredentials(credentialsProvider().getCredentials()).build().getService();
```

Comment:
do you want/need to always create a new client or can the client be reused? Something like [this](https://github.com/tnb-software/TNB/blob/main/system-x/services/google/cloud/storage/src/main/java/software/tnb/google/storage/service/GoogleStorage.java#L37-L47)